cmd/gateway/zerostor/README.md

```diff
@@ -170,4 +156,68 @@ Check the downloaded file is not corrupt
 cmp minio /tmp/bin-minio
 ```
 
+## Healer API
```

Comment:
Can we also add an example client call to the healing API that steams the result. in GO and/or Python ?

Reply:
Sure